edx_username_changer/admin.py

```diff
@@ -0,0 +1,36 @@
+"""
+Django admin pages for edx-username-changer plugin
+"""
+
+from django.conf import settings
+from django.contrib import admin
+from django.contrib.auth import get_user_model
+
+from common.djangoapps.student.admin import UserAdmin as BaseUserAdmin
+
+
+User = get_user_model()
+
+
+class UserAdmin(BaseUserAdmin):
+    """
+    Admin interface for the User model.
+    """
+
+    def get_readonly_fields(self, request, obj=None):
+        """
+        It will remove 'username' from readonly field to make it editable through admin panel
+        """
+        readonly_fields = super(UserAdmin, self).get_readonly_fields(request, obj)
```

Comment:
```suggestion
        readonly_fields = super().get_readonly_fields(request, obj)
```

I think we can use the super directly with python3 owards.

Reply:
updated.